polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,264 @@
+
+Crowdloans enabled users to securely lend funds to teams for lease deposits in exchange for pre-sale tokens, which is the only way to access slots on Polkadot 1.0.
+
+!!!note
+    Auction are deprecated in favor of Coretime
+
+Additionally, the  chain handles [staking](add link to PoS page){target=\_blank}, [accounts](add a link to accounts pages){target=\_blank}, balances, and [governance](add a link to govenance page){target=\_blank}.
+
+#### Agile Coretime
+
+The new and more efficient way of obtaining core on Polkadot is to go through process of purchasing a **coretime**.
+
+[**Agile Coretime**](add link to Agile Coretime page) improves the efficient use of Polkadot's network resources and offers economic flexibility for developers, extending Polkadot's capabilities far beyond the original vision outlined in its [whitepaper](https://polkadot.com/papers/Polkadot-whitepaper.pdf){target=\_blank}.
+
+It enables parachains to purchase monthly "bulk" allocations of **coretime**(The time allocated for utilizing a core, measured in Polkadot relay chain blocks), ensuring heavy-duty parachains that can author a block every 6 seconds with [Asynchronous Backing](add link to async backing page) can reliably renew their coretime each month. Although six second block times is now the default, parachains have the option of producing blocks less frequently. 
+
+Renewal orders are prioritized over new orders, offering stability against price fluctuations and helping parachains budget more effectively for their project costs.
+
+### Polkadot's Resilience
+
+Decentralization is a vital component of blockchain networks, but it comes with trade-offs:
+
+- An overly decentralized network may face challenges in reaching consensus and require significant energy to operate
+- Also a network that achieves consensus quickly risks centralization, making it easier to manipulate or attack
+
+Ideally, a network should be decentralized enough to prevent any manipulative or malicious influence. In this sense, decentralization serves as a tool to achieve resilience. 
+
+**Polkadot 1.0** currently achieves resilience through several strategies:
+
+1. **Nominated Proof of Staking (NPoS)** - this ensures that the stake per validator is maximized and evenly distributed among validators
+
+2. **Decentralized Nodes** - designed to encourage operators to join the network, this program aims to expand and diversify the validators in the ecosystem that aim to become independent of the program during their term. Feel free to explore more about the program on the official [Decentralized Nodes](https://nodes.web3.foundation/){target=\_blank} page.
+
+3. **On-Chain Treasury and Governance** - known as [OpenGov](link to opengov overview page){target=\_blank}, this system allows every decision to be made through public referenda, enabling any token holder to cast a vote
```

Comment:
```suggestion
Decentralization is a vital component of blockchain networks, but it comes with trade-offs:

- An overly decentralized network may face challenges in reaching consensus and require significant energy to operate
- Also, a network that achieves consensus quickly risks centralization, making it easier to manipulate or attack

A network should be decentralized enough to prevent manipulative or malicious influence. In this sense, decentralization is a tool for achieving resilience. 

Polkadot 1.0 currently achieves resilience through several strategies:

- **Nominated Proof of Stake (NPoS)** - this ensures that the stake per validator is maximized and evenly distributed among validators

- **Decentralized nodes** - designed to encourage operators to join the network. This program aims to expand and diversify the validators in the ecosystem who aim to become independent of the program during their term. Feel free to explore more about the program on the official [Decentralized Nodes](https://nodes.web3.foundation/){target=\_blank} page

- **On-chain treasury and governance** - known as [OpenGov](link to opengov overview page){target=\_blank}, this system allows every decision to be made through public referenda, enabling any token holder to cast a vote
```